### ISSUE TITLE
Schema creation for proxy-spanner broken

### ISSUE THREAD

**Issue by hseeberger** [cloudstateio/cloudstate#483]
```
io.grpc.StatusRuntimeException: INVALID_ARGUMENT: Error parsing Spanner DDL statement: CREATE TABLE journal (\n  persistence_id STRING(MAX) NOT NULL,\n  sequence_nr INT64 NOT NULL,\n  event BYTES(MAX),\n  ser_id INT64 NOT NULL,\n  ser_manifest STRING(MAX) NOT NULL,\n  write_time TIMESTAMP NOT NULL OPTIONS (allow_commit_timestamp=true),\n  writer_uuid STRING(MAX) NOT NULL,\n) PRIMARY KEY (persistence_id, sequence_nr);\nCREATE TABLE tags (\n  persistence_id STRING(MAX) NOT NULL,\n  sequence_nr INT64 NOT NULL,\n  tag STRING(MAX) NOT NULL,\n  write_time TIMESTAMP NOT NULL OPTIONS (allow_commit_timestamp=true),\n) PRIMARY KEY (persistence_id, sequence_nr, tag),\nINTERLEAVE IN PARENT journal ON DELETE CASCADE;\nCREATE TABLE deletions (\n  persistence_id STRING(MAX) NOT NULL,\n  deleted_to INT64 NOT NULL,\n) PRIMARY KEY (persistence_id);\nCREATE INDEX tags_tag_and_offset\nON tags (\n  tag,\n  write_time\n);\nCREATE TABLE snapshots (\n  persistence_id STRING(MAX) NOT NULL,\n  sequence_nr INT64 NOT NULL,\n  timestamp TIMESTAMP NOT NULL,\n  ser_id INT64 NOT NULL,\n...
```

**Comment by pvlugter** [comment]
Added issue to see if we can run integration tests for proxy-spanner too: #487

**Comment by pvlugter** [comment]
Should be fixed by #485
